showcase/src/assets-base/stark_logo_black.svg

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
```

Comment:
The logos should not be included in the showcase, they should be part of `stark-core` only... check my other remark